scarlet/observation.py

```diff
@@ -146,11 +152,32 @@ def match(self, scene):
                 new_kernel_fft.append(np.fft.fft2(np.fft.ifftshift(kernel)))
             self.psfs_fft = np.array(new_kernel_fft)
 
-        # 4) divide obs.psf from scene.psf in Fourier space
+        return self
 
-        # [ 5) compute sparse representation of interpolation * convolution ]
+    @property
```

Comment:
Again, why are these needed? `Observation` *is* a `Scene`, so these methods are fully redundant. Even if we change it so that `Observation` has a `Scene`, you can always call these methods of the class member `scene`. Please remove

Reply:
same as before